packages/taquito-rpc/src/rpc-client-modules/rpc-cache.ts

```diff
@@ -798,6 +800,17 @@ export class RpcClientCache implements RpcClientInterface {
     return this.rpcClient.runCode(code, { block });
   }
 
+  /**
+   * @param viewParams Parameters of the view to run
+   * @param options contains generic configuration for rpc calls
+   *
+   * @description Simulate a call to a view following the TZIP-4 standard. See https://gitlab.com/tzip/tzip/-/blob/master/proposals/tzip-4/tzip-4.md#view-entrypoints.
+   *
+   */
+   async runView(viewParams: RPCRunViewParam, { block }: RPCOptions = defaultRPCOptions): Promise<RunViewResult> {
```

Comment:
consider adding a default value for unparsing_mode as in the RpcClient class. 
same for the rpc-wrapper